### PR TITLE
Updates UFO-S ontology (nardi2015commitment)

### PR DESCRIPTION
**Pull request by claudenirmf**
This pull requests updates documentation of the UFO-S ontology.

This PR is partially solves #1 . 

Additionally, it also includes minor fixes to the repository's README file and useful updates to the .gitignore.

### CODE REVIEW

**Review by Matt-81 — COMMENTED** [review]
I would make more explicit the mapping between original and output diagrams. We have 13 files as original diagrams and 3 files as new diagrams.

Did you use the information from all the 13 inputs to generate the 3 final outputs?

**Review by Matt-81 — COMMENTED** [review]
Your new diagrams look very cool =) Maybe we can provide some more constraints on how to design the diagrams. Now we just say to stick with the original diagram as much as possible, but we may add some info like the ones you sent me (e.g., paint through labels, orthogonal vs. organic, ...)

**Review by Matt-81 — CHANGES_REQUESTED** [review]
Maybe in the .bib is better to arrange the refs in chronological order. Or maybe you arranged them according to different criteria?

**Comment by claudenirmf** [comment]
> Did you use the information from all the 13 inputs to generate the 3 final outputs?

I have used the diagram number to identify the relationship between original and old diagrams.

> Maybe we can provide some more constraints on how to design the diagrams.

Sure, let's discuss that on the meeting

> Maybe in the .bib is better to arrange the refs in chronological order.

I don't think this makes any difference since the file only contains a few reference. But I'm open to be convinced that this needs to be enforced.

File: nardi2015commitment/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+title: UFO-S
```
Comment (claudenirmf) [inline nardi2015commitment/metadata.yml:1]: This is not the correct name

**Comment by claudenirmf** [comment]
Remember to use "Squash and Merge" when accepting this PR.

Screen Shot 2021-11-15 at 16 50 56

**Pull request merged**